src/foam/nanos/script/Script.js

```diff
@@ -198,7 +200,8 @@ foam.CLASS({
       }
```

Comment:
is max output length still necessary to limit? since it's all in memory now it shouldn't matter.

Reply:
If I remember correctly, I believe this max output has more to do with HTTP limits rather than memory/disk concerns.